### PR TITLE
Split 3.x hosting docs off, create 4.x hosting section

### PR DESCRIPTION
**Pull request by mrjones-plip**
This does all the foundational work to split up hosting documents in prep  for 4.0.  It only updates the contents of the "Local Setup" document for 4.x.  The rest of the 4.x content will need to still be written.

**NB** - before merging this branch, we need to merge medic/cht-core/pull/7861

Changes:
* created a 3.x and 4.x hosting directory
* moved these files into 3.x:
   * app-developer.md
   * ec2-setup-guide.md
   * offline.md
   * self-hosting.md
   * ssl-cert-install.md
* kept these files, but updated them to denote the apply to both 3.x and 4.x:
   * core-developer.md
   * monitoring.md
   * requirements.md
* all 3.x pages that were moved got an alias, so going to the URL w/ out the `/3.x/` in it, will redirect you 
   * app-developer.md
   * ec2-setup-guide.md
   * offline.md
   * self-hosting.md
   * ssl-cert-install.md
* updated and tested tutorials/local-setup.md to be for 4.x,

### CODE REVIEW

**Review by esthermmoturi — APPROVED** [review]

File: content/en/apps/tutorials/local-setup.md [hunk 1]
```diff
@@ -172,19 +180,18 @@ With the blank project deployed to your CHT instance, you're ready to start writ
 To install a valid certificate, open a terminal in the `cht-core` directory. Ensure the `medic-os` container is running and make this call:
```
Comment (garethbowen) [inline content/en/apps/tutorials/local-setup.md:180]: We should drop references to `medic-os` while we're here.

```suggestion
To install a valid certificate, open a terminal in the `cht-core` directory. Ensure the CHT instance is running and make this call:
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
Great first step, thanks! A couple of suggestions inline.

File: content/en/apps/guides/updates/preparing-for-4.md [hunk 2]
```diff
@@ -157,7 +159,8 @@ CHT 4.0 [upgrades the version of Enketo](https://github.com/medic/cht-core/pull/
 
 You can also manually test your forms on a non-prod CHT instance. It is possible to test your forms against the new Enekto changes without having to uplift your non-prod CHT instance to the new 4.0 architecture.
 
-An easy way of doing this is to use the [CHT Docker Helper]({{< relref "apps/guides/hosting/app-developer#cht-docker-helper" >}}) to deploy a 3.x CHT instance. After you have your dev instance up and running, upgrade to the `enketo_upgrade_3.x` branch:
+An easy way of doing this is to use the [CHT Docker Helper]({{< relref "apps/guides/hosting/3.x/app-developer#cht-docker-helper" >}}) to deploy a 3.x CHT instance. After you have your dev instance up and running, upgrade to the `7786-fix-report-label` branch:
+An easy way of doing this is to use the [CHT Docker Helper]({{< relref "apps/guides/hosting/3.x/app-developer#cht-docker-helper" >}}) to deploy a 3.x CHT instance. After you have your dev instance up and running, upgrade to the `enketo_upgrade_3.x` branch:
```
[the remark this answers is not in the document]
Comment (mrjones-plip) [reply content/en/apps/guides/updates/preparing-for-4.md]: D'oh! Fixed in 458dee8

**Review by garethbowen — APPROVED** [review]
Awesome, thanks.

**Pull request merged**